### ISSUE TITLE
Auto-backup to Nextcloud

### ISSUE THREAD

**Issue by kaibagley** [ImranR98/Obtainium#1267]
**Describe the bug**
Unable to set backup location to a Nextcloud folder, which shows up on the Android file browser when the Nextcloud Android client is installed.

**To Reproduce**
1. Click "Pick Export Directory"
2. Choose a directory in Nextcloud
3. No response, where usually a banner appears along the bottom to say the backup was created

**Please complete the following information:**
 - Device: Pixel 7 Pro
 - OS: GraphenoOS / AOSP 14
 - Obtainium Version: 0.15.8-beta

**Comment by ImranR98** [comment]
I can't reproduce this as I don't see a NextCloud icon (probably a Google Files feature). Could you send a screenshot?

**Comment by kaibagley** [comment]
I'm using the AOSP files app, default files app that comes with GrapheneOS. Here's a screenshot of the files app with Nextcloud

![IMG_20240111_104146](https://github.com/ImranR98/Obtainium/assets/53884957/26aabfd4-e2f1-41b9-80fa-a505106ba526)

**Comment by ImranR98** [comment]
Hmm, for some reason the plugin we use to share files just doesn't return any result when sharing to the NextCloud directory, there's probably some error on the Android side that doesn't bubble up. For now I'd recommend just exporting to a local directory.

**Comment by kaibagley** [comment]
Yeah my current workaround is just to export locally, then move that to Nextcloud. Would you like me to keep this open until it is solved? Or close since it may not be on Obtainium's side

**Comment by rm76** [comment]
I can confirm this issue too.
I, too, am on Graphene OS.
Exporting to a Nextcloud folder fails silently.
Exporting to a local folder works just fine.

**Comment by Welved** [comment]
Yea I am having the same issue. I can go and set the export directory and everything, it just doesn't do anything.

I tapped the 'Obtainium Export' button, it legitimately does nothing; didn't even write anything to my nextcloud.

I've even toggled on the 'auto-export on changes' and 'include settings', that also does nothing.

Device: Pixel 6a
OS: CalyxOS / Android 14
Obtainium Version: 1.1.31

**Comment by Welved** [comment]
> Yeah my current workaround is just to export locally, then move that to Nextcloud. Would you like me to keep this open until it is solved? Or close since it may not be on Obtainium's side

How do you go about doing a local export? There is no option in my copy of obtainium to do so and it doesn't my phone's internal storage doesn't have a directory for obtainium to obtain any amount of app data.
